diff --git a/runtime/node/process/index.mjs b/runtime/node/process/index.mjs
index 9695596b8e8c88230921998d86be0b8c45fc59cf..d185f9edb14a7a7c0a4efcd083efdc06077c94fc 100644
--- a/runtime/node/process/index.mjs
+++ b/runtime/node/process/index.mjs
@@ -1,2 +1,4 @@
 import { process as unenvProcess } from "./internal/process.mjs";
 export default unenvProcess;
+export const stdin = unenvProcess.stdin
+export const stdout = unenvProcess.stdout
\ No newline at end of file
diff --git a/runtime/polyfill/process.mjs b/runtime/polyfill/process.mjs
index ca819cbd9a4996c02e3e7e951a4bdb5bd29e6bf7..a36412484f9498e28eaa9614c7b91bb56c52e035 100644
--- a/runtime/polyfill/process.mjs
+++ b/runtime/polyfill/process.mjs
@@ -1,5 +1,7 @@
 import unenvProcess from "../node/process/index.mjs";
 Object.assign(globalThis.process, {
   ...unenvProcess,
-  ...globalThis.process
+  ...globalThis.process,
 });
+
+export default unenvProcess
